statement.dd

```diff
@@ -206,7 +206,7 @@ $(H3 $(LEGACY_LNAME2 DeclarationStatement, declaration-statement, Declaration St
 
 $(GRAMMAR
 $(GNAME DeclarationStatement):
-    $(GLINK2 declaration, Declaration)
+    $(GLINK2 declaration, StorageClasses)$(OPT) $(GLINK2 declaration, Declaration)
```

Comment:
StorageClasses should not link to #declaration. Should be: `$(GLINK StorageClasses)`